README.md

```diff
@@ -3,21 +3,132 @@
+
+Any values left empty while using the MLSpace Config Wizard will default to what is set in the `lib/constants.ts` file.
+
+The MLSpace Config Wizard can be invoked with the command:
+
+```Bash
+npm run config
+```
+
+This will prompt you to choose between Basic Config and Advanced Config.
+
+- Basic Config - only prompts for the properties which must be set in order to deploy MLSpace.
+- Advanced Config - prompts for required fields as well as optional configurations which are commonly customized.
+
+If selecting Basic Config, the properties you will be prompted for are:
+
+- AWS account ID: the AWS account ID for the account MLSpace will be deployed into
+- AWS region: the region that MLSpace resources will be deployed into
+- OIDC URL: the OIDC endpoint that will be used for MLSpace authentication
+- OIDC Client Name: the OIDC client name that should be used by MLSpace for authentication
+
+If selecting Advanced Config you will be prompted for the same properties Basic Config prompts for, as well as other optional values. Anything not specific will use the defaults in `constants.ts` and/or provisioned by MLSpace. 
+
+The Advanced Config will ask:
+
+**Do you want to use existing VPC?**
+If you answered yes you will be prompted for:
+
+- VPC Name: if MLSpace is being deployed into an existing VPC this should be the name of that VPC
+- VPC ID: if MLSpace is being deployed into an existing VPC this should be the ID of that VPC
+- VPC Default Security Group: if MLSpace is being deployed into an existing VPC this should be the default security group of that VPC
+
+**Do you want to use existing IAM Roles?**
+If you answered yes you will be prompted for:
+
+- S3 Reader Role ARN: ARN of an existing IAM role to use for reading from the static website S3 bucket
+- Bucket Deployment Role ARN: ARN of an existing IAM role to use for deploying to the static website S3 bucket
+- Notebook Role ARN: ARN of an existing IAM role to associate with all notebooks created in MLSpace
+- App Role ARN: ARN of an existing IAM role to use for executing the MLSpace lambdas
+- EMR Default Role ARN: ARN of an existing IAM role that will be used as the 'ServiceRole' for all EMR clusters
+- EMR EC2 Instance Role ARN: ARN of an existing role that will be used as the 'JobFlowRole' and 'AutoScalingRole' for all EMR clusters
+
+**Do you want to modify the banner displayed on MLSpace?**
+If you answered yes you will be prompted for:
+
+- System Banner Text: the text to display on the system banner displayed at the top and bottom of the MLSpace web application. If set to a blank string no banner will be displayed
+- System Banner Background Color: the background color of the system banner if enabled
+- System Banner Text Color: the color of the text displayed in the system banner if enabled
+
+### Option 2 - configure by updating lib/constants.ts
```

Comment:
I would personally capitalize these section names: "Configure by updating..."

Not blocking